config/migration/migration_types/types.go

```diff
@@ -0,0 +1,16 @@
+package migration_types
+
+type Migratable interface {
+	Save(filename string) error
+	ToNext() (Migratable, error)
+}
+
+type NullConfig struct{}
```

Comment:
```suggestion
type EmptyConfig struct{}
```

Reply:
In this case, `NullConfig` serves to indicate that the config does not exist or is invalid, ergo "Null". "Empty" has a different meaning. Empty configs are also possible in any of the already existing Config types.